frontend/src/index.css

```diff
@@ -102,6 +109,10 @@ input[type="number"] {
   margin-left: 1rem;
 }
 
+#dropdown-button {
+  margin-right: 1rem;
+}
+
```

Comment:
You don't need this if you modify the `.basic-button` class slightly. (see comment above).
It's shorter, and afaik it's also considered rather bad practice to use the id to assign styles. Rule of thumb is: id is for functionality, class is for styling.